.storybook/main.js

```diff
@@ -1,8 +1,11 @@
 module.exports = {
   stories: [
-    '../src/stories/*.stories.mdx',
-    '../src/stories/*.stories.@(js|jsx|ts|tsx)',
-    '../src/stories/*/*.stories.@(js|jsx|ts|tsx)',
+    // '../src/stories/*.stories.mdx',
+    // '../src/stories/*.stories.@(js|jsx|ts|tsx)',
+    // '../src/stories/*/*.stories.@(js|jsx|ts|tsx)',
+    './stories/*.stories.mdx',
```

Comment:
Won't this break? The stories folder is also in the `src` folder

Reply:
Just noticed you moved all the files into the .storybook folder. Is this recommended somewhere? I'm not sure whether we want to introduce this change into the project structure like this.

Reply:
The reason for this is now explained in the PR description, namely that every story requires access to typescript files to generate documentation.  In our build we no longer expose ts files, furthermore the storybook file watchers were not working anymore, and storybook was not reloaded on updates in stories.tsx or component source code.